contrib/guix/libexec/build.sh

```diff
@@ -208,6 +208,8 @@ mkdir -p "$DISTSRC"
     # Build Bitcoin Core
     make --jobs="$MAX_JOBS" ${V:+V=1}
 
+    # Check that ELF security checks tools are sane
```

Comment:
nit: could drop `ELF` from here and below. As the security (and symbol) checks are across all platforms.

Reply:
Fixed!